.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,45 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the
+// README at: https://github.com/devcontainers/templates/tree/main/src/ubuntu
+{
+  "name": "Station Data Portal",
```

Comment:
This should be "Climate Explorer" instead.

Reply:
Good call, now fixed.